platform-hub-api/spec/controllers/contact_lists_controller_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'rails_helper'
+
+RSpec.describe ContactListsController, type: :controller do
+
+  describe 'GET #show' do
+    before do
+      contact_list_hash_record = create :contact_list_hash_record
+      @contact_list = ContactList.new contact_list_hash_record
```

Comment:
May not be able to use `let` here since those are lazily evaluated (well, unless I use `let!`). Generally, I prefer `let`'s to be very simple data creation / test data, so this is a potential candidate, but since I needed intermediate state I figured we stick to the convention of instantiated the "primary" object in the `before` block and assigning to an instance variable (like we do for other controller spec cases).